keel/api/permissions.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import permissions
+from django.conf import settings
+
+
+class CustomLeadPermission(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+
+        header = request.headers['X-CLIENT_ID']
```

Comment:
do a get method instead...else it will throw error at runtime....and that way you can check first if there is header present and then its value

Reply:
updated